examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
> I think it will also be needed to patch CMakeLists.txt of zmqpp to remove the lines at 

Yes. Again, at least under condition `if(no_toolchain)`.

> I guess when hunter moves to CMake 3.1, ideally we would add

I think this CMake feature leads to conflicts in some cases. Setting standard version locally either by modifying `CMAKE_CXX_FLAGS` or by `CXX_STANDARD` is a bad design. The reason is a compatibility of dependencies. If project `Foo` set `CXX_STANDARD 98` and `Boo` set `CXX_STANDARD 11` and both of them depends on header-only Boost then for `Foo` and `Boo` different incompatible structures/methods will be generated from Boost. The only way to fix this I see in setting it globally in toolchain (by `CXX_STANDARD` or by `CMAKE_CXX_FLAGS`).